kernel/src/table_changes/physical_to_logical.rs

```diff
@@ -124,7 +124,7 @@ mod tests {
                 Scalar::Long(20).into(),
                 expected_expr,
                 Expr::literal(42i64),
-                Scalar::TimestampNtz(1234000).into(), // Microsecond is 1000x millisecond
+                Scalar::Timestamp(1234000).into(), // Microsecond is 1000x millisecond
```

Comment:
given that we only had to update a test that checks the physical-to-logical expression, this points to a test gap on the actual schema of the returned batches. perhaps we should have a simple test that just asserts the data types of whatever is returned? (this is somewhat tested here but I don't believe it's fully covered - if there was e.g. an incorrect schema application _after_ the expression, this test would pass but the new "schema test" would fail)

Reply:
Added a check in the e2e test.